### ISSUE TITLE
As the Rogue River-Siskiyou Open Forest POC, I want to review and have Christmas tree permit content updated so that I know the information is correct before the season starts.

### ISSUE THREAD

**Issue by lisamcmichael** [USDAForestService/fs-open-forest-platform#803]
## Notes
See EPIC #799
Word doc on Pinyon: https://usfs.box.com/s/019i5ky69r0bfc7myhsmd8pbb9rfvtyf

## Acceptance Criteria
- [x] POC reviewed and approved content
- [x] POC approved any backlog content ( if there is any for next season)
- [x] The Forest's content has been updated based on Forest POC review.

## Tasks
- [x] Start scheduling content review with POC
- [x] Complete a demo with the forest, captured input
- [x] Review findings with PO

- [x] **Implement changes:**
- [x] Throughout the site "National" is missing from the Forest's name. Include "National" in every reference to the forest as: Rogue River-Siskiyou National Forest (including breadcrumb trail). There are 6 instances.
-  [x] SEASON DATES: Nov 8, 2019 - Dec 25, 2019
- [] SEASON DATES: Nov 9, 2019 - Dec 25, 2019
- [x] WHERE TO CUT YOUR TREE (**under Prohibited areas...**), Revise "Cut your Christmas tree only within national forest boundary" to read as follows: "Cut trees only within the national forest boundary and respect surrounding private property."

CUTTING AREA MAPS: 
Christmas tree maps are provided at the time of permit purchase and [available online.](https://www.fs.usda.gov/detail/rogue-siskiyou/home/?cid=fseprd563908)

- [x] [Click to view map of Grants Pass East in Avenza](https://www.avenzamaps.com/maps/850984/rogue-river-siskiyou-nf-grants-pass-east-christmas-tree-harvest-map) 

- [x] [Click to view map of Grants Pass West in Avenza](https://www.avenzamaps.com/maps/850985/rogue-river-siskiyou-nf-grants-pass-west-christmas-tree-harvest-map) 

Also, these maps are available for FREE via Avenza! Download the app on your smartphone to help guide where to cut your tree!  

- [x] ADD text: Christmas Tree map of Medford North/South cutting areas. (Click to view larger map) and ADD MAP https://www.fs.usda.gov/Internet/FSE_DOCUMENTS/fseprd604596.pdf

- [x] ADD text: Christmas Tree map of Grants Pass East/West cutting areas. (Click to view larger map) and ADD MAP  https://www.fs.usda.gov/Internet/FSE_DOCUMENTS/fseprd604591.pdf

- [x] Remove QR Code Image file from page.

- [x] HOW TO CUT YOUR TREE/ HELPFUL CUTTING TIPS: Second bullet point (revise "8 feet" to "12 feet". Example: "Be sure another tree has space to grow within 12 feet of  the one you plan to cut."
- [x] HOW TO CUT YOUR TREE/IDENTIFYING TREE SPECIES: The forest wants to include a _new_ tree species "Shasta Fir". List this Shasta Fir first, followed by Douglas-fir and Pacific Silver Fir. **Remove Noble Fir.**  
- [x] **FOLLOW UP** on image for Shasta Fir and bullet points
• Needles are generally 0.75- 1.5 inches long
• Mature needles are dark blue-green. Newer growth is a silvery color
• Has stiff strong branches, great for heavier ornaments
• Holds needles and maintains color well

We’ll use this image
https://plants.usda.gov/java/largeImage?imageID=abma_004_ahp.jpg

- [x] HOW TO PLAN YOUR TRIP: The first bullet point (**Add**): Do not park at a Sno-Park unless you have a valid Sno-Park permit displayed in your vehicle.
- [x] CONTACT US: **Revise** the first paragraph, the last sentence, to read: All offices are closed on the weekends and federal holidays.
- [] CONTACT US: **FOLLOW-UP** - Forest has an updated PDF for "local vendors" hyperlink. 
- [x] CONTACT US: For each location, spell out "RD" as Ranger District.
- [x] CONTACT US: For each location, remove Fax no#s. 
- [x] CONTACT US: Remove "J. Herbert Stone Nursery" address info
- [x] CONTACT US: Add the following (after) all addresses at the bottom of the page: Visit [Rogue River-Siskiyou National Forest](https://www.fs.usda.gov/rogue-siskiyou/) for more information.

- [x] ALT Text and Tags added to images (provided by POC)
- [x] Forest POC has been informed of updates
- [x] PO approved

## Definition of Done
- [x] Pull requests meet technical definition of done
- [ ] Compare finished design with mockup
- [ ] Usability tested with forest

**Comment by lisamcmichael** [comment]
@carlsonem and @ASprinkle The content review is scheduled for 9/4.

**Comment by ASprinkle** [comment]
Robert Barnhart is the POC for the RRS. He has accepted a new position (in Region 6) and will be starting that new position early October.

**Comment by lisamcmichael** [comment]
Bob invited Anna McGovern to the content review on Wed. 9/4. anna.mcgovern@usda.gov

**Comment by lisamcmichael** [comment]
Completed RRS content review

**Comment by lisamcmichael** [comment]
@ASprinkle I have updated the story with the Forest's changes.  There is one follow-up and backlog item noted above.

**Comment by lisamcmichael** [comment]
**Word document of Content Review on Box:**
https://usfs.box.com/s/019i5ky69r0bfc7myhsmd8pbb9rfvtyf

**Comment by lisamcmichael** [comment]
@mtlaney @briandavidson I have some follow-ups; I'll get those to you asap.

**Comment by lisamcmichael** [comment]
Moved to Blocked, several forest followups

**Comment by ASprinkle** [comment]
Also waiting for the Forest to provide more information regarding Service First applicability

**Comment by ASprinkle** [comment]
Received an update from Robert today on the phone:

- They are using the same maps this year as last year
- Having discussions with BLM regarding Service First (BLM State Rep position is currently vacant) Scott Loose is the BLM POC that he is coordinating with (Phone # TBD).
- Rob is scheduling a presentation for Open Forest on Tues 9/17 (time TBD) for the Forest's leadership team.  
-We will continue forward with permits valid on the RRS Forest only at this time, until we hear differently from the State Representatives (this is contradictory to the information on the maps however, which allow cutting the green spaces
-Rob is transitioning to a new position. Anna McGovern and Chamise Kramer will be the POC for the RRS after Rob transitions.

**Comment by ASprinkle** [comment]
•	Needles are generally 0.75- 1.5 inches long
•	Mature needles are dark blue-green. Newer growth is a silvery color
•	Has stiff strong branches, great for heavier ornaments
•	Holds needles and maintains color well

We’ll use this image
https://plants.usda.gov/java/largeImage?imageID=abma_004_ahp.jpg

**Comment by ASprinkle** [comment]
@lisamcmichael I am "unblocking" this story based on the updates from Robert. I am moving to Ready for dev..

**Comment by lisamcmichael** [comment]
I sent a note to the Forest to request this info:
_CONTACT US: **FOLLOW-UP** - Forest has an updated PDF for "local vendors" hyperlink._

**Comment by ASprinkle** [comment]
Thanks @lisamcmichael Hopefully when they updated their PDF on their website the link we're using doesn't change. We can check that later. I'd say we can consider the follow up for this task completed in the task list above.

**Comment by briandavidson** [comment]
@lisamcmichael and @ASprinkle - there are a few remaining tasks for adding the cutting area maps, but the links provided are for pdf files. Do we have actual image files for these maps, or are we just linking to the PDF files by having the users click on the text, i.e.: "Christmas Tree map of Medford North/South cutting areas. (Click to view larger map)" ?

**Comment by ASprinkle** [comment]
@briandavidson I do not think we have actual image files for the maps - but I defer to @lisamcmichael

**Comment by lisamcmichael** [comment]
@briandavidson We do not have image files.

**Comment by ASprinkle** [comment]
@briandavidson  since we don't have image files, let's do the text hyperlink description without any image files per the task list above:

ADD text: Christmas Tree map of Medford North/South cutting areas. (Click to view larger map) and ADD MAP https://www.fs.usda.gov/Internet/FSE_DOCUMENTS/fseprd604596.pdf

 ADD text: Christmas Tree map of Grants Pass East/West cutting areas. (Click to view larger map) and ADD MAP https://www.fs.usda.gov/Internet/FSE_DOCUMENTS/fseprd604591.pdf

Also, Remove the QR codes. (Added a task for this above) We've got tasks for including text hyperlinks to Avenza.

**Comment by ASprinkle** [comment]
I reviewed this morning - It doesn't look like the Where to cut your tree section items have been updated in staging yet. Also, I don't see the updates in the Identifying tree species section.

**Comment by mtlaney** [comment]
@ASprinkle @lisamcmichael @briandavidson I can convert the pdf's into png image files and load them correctly on the site.

**Comment by lisamcmichael** [comment]
@mtlaney Welcome back 👍

**Comment by ASprinkle** [comment]
@mtlaney @briandavidson The RRS just sent me an email informing me that their season start date is now 
### Nov. 8.
I updated the tasks list above...  can we update this on the site in this story or should we address it separately?

**Comment by mtlaney** [comment]
new date of Nov 8th is now set

**Comment by mtlaney** [comment]
I didn't see any alt text anywhere from the POC is that correct? I've added basic alt text saying what map it is for the time being.

**Comment by ASprinkle** [comment]
@mtlaney ok let's go with what you've done for now. We can add a story later to update the Alt Text once we find it.

**Comment by ASprinkle** [comment]
These updates look great. I noticed we never got the follow up on the Local Vendors list. We can address that in a separate story, and maintain the same link.  Ideally, when the forest updates the PDF info on their website, it should reference the same link.

**Comment by lisamcmichael** [comment]
@mtlaney I dropped them a note just now to nudge them about the Local Vendors list and link, if it's to be changed from what they currently have: https://www.fs.usda.gov/Internet/FSE_DOCUMENTS/fseprd602895.pdf
